src/js/svgo-worker/index.js

```diff
@@ -10,10 +10,13 @@ const createDimensionsExtractor = () => {
         element: {
           enter: ({ name, attributes }, { type }) => {
             if (name === 'svg' && type === 'root') {
-              if (attributes.width != null && attributes.height !== null) {
+              if (
+                attributes.width !== undefined &&
```

Comment:
This one is more verbose, happy to go with `!= null` for both checks; previously one of them used the strict comparison which might have been wrong